tla/consensus/ccfraft.tla

```diff
@@ -1224,13 +1229,14 @@ MonoLogInv ==
 LogConfigurationConsistentInv ==
     \A i \in Servers :
         \/ state[i] = None
+        \* Follower, but no known configurations yet
+        \/ /\ state[i] = Follower
+           /\ Cardinality(DOMAIN configurations[i]) = 0
         \/
             \* Configurations should have associated reconfiguration txs in the log
             /\ \A idx \in DOMAIN (configurations[i]) : 
                 /\ log[i][idx].contentType = TypeReconfiguration            
                 /\ log[i][idx].configuration = configurations[i][idx]
-            \* Current configuration should be committed
```

Comment:
could this be either configurations is empty or current configurations is before the commit index?

Reply:
Empty when Follower is dealt with on line 1232, but this just generally wrong as soon as there is a new pending configuration, since CurrentConfigurationIndex is just whatever the latest known config is. It may be not even be committable at all if a signature has not been emitted.

I am not sure what, if anything, we want to assert here.